Source/WebKit/WebKitSwift/LinearMediaKit/WKSLinearMediaTypes.h

```diff
@@ -98,6 +94,19 @@ API_AVAILABLE(visionos(1.0))
 @property (nonatomic, readonly) float disparityAdjustment;
 @end
 
-NS_ASSUME_NONNULL_END
+NS_SWIFT_UI_ACTOR
+@interface WKSPlayableViewControllerHost : NSObject
+
+@property (nonatomic, readonly) UIViewController *baseViewController;
```

Comment:
Would be OK to just call this `viewController` or `playableViewController` now that the class has a better name.